steward/src/commands/tx/cosmos.rs

```diff
@@ -25,9 +25,13 @@ impl Runnable for Cosmos {
     }
 }
 
+/// Send tx from Cosmos to Ethereum chain
 #[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Send transactions from Cosmos to Ethereum chain.\n This command sends tx from the Cosmos chain to the Eth chain.\n It takes a free vector of strings as required flags for Cosmos key, Eth key and amount."
+)]
 pub struct SendToEth {
-    free: Vec<String>,
+    required_flags: Vec<String>,
```

Comment:
If we are requiring flags, rather than using a vector of strings, we should create elements inside this struct for each flag with no default value.

Reply:
Following up on this comment as you addressed the "required args" format in most of the other commands but not this one. Alternatively, if these are duplicate commands that are unfinished, perhaps we should simply delete the `tx` subcommand structure entirely.